staging/src/k8s.io/apiserver/pkg/util/flowcontrol/borrowing_test.go

```diff
@@ -150,19 +150,17 @@ func TestBorrowing(t *testing.T) {
 			})
 
 			ctx, cancel := context.WithTimeout(context.Background(), 50*time.Second)
-			stopCh := ctx.Done()
-			controllerCompletionCh := make(chan error)
-
-			informerFactory.Start(stopCh)
+			defer cancel()
```

Comment:
In unit tests use `_, ctx := ktesting.NewTestContext(t)` to get per-test output.

Reply:
done